src/pages/Missions/Missions.tsx

```diff
@@ -46,11 +46,18 @@ export const useGetMissions = () => {
   const translate = useTranslate()
   const missionItems: MissionProps[] = useMemo(() => {
     return [
+      // {
+      //   title: translate('missions.foxatar.title'),
+      //   subtitle: translate('missions.foxatar.subtitle'),
+      //   buttonText: translate('missions.foxatar.cta'),
+      //   coverImage: FoxAtarBg,
+      //   onClick: () => window.open('https://app.mercle.xyz/shapeshift/events'),
+      // },
```

Comment:
gonna request changes on your first PR 🤣

can we please delete rather than comment out code?

Reply:
second PR 😉 

yeah i'm cool with delete.